### PR TITLE
007 authorisation

### PR DESCRIPTION
**Pull request by TaylorB-softdevkainos**

### CODE REVIEW

**Review by ldevlin559 — APPROVED** [review]

File: src/services/JobRoleService.ts [hunk 1]
```diff
@@ -18,14 +19,14 @@ export const getJobs = async (): Promise<JobRole[]> => {
...
 }
 
-export const getJobDetailsById = async (id: string): Promise<JobRoleDetailedResponse[]> => {
+export const getJobDetailsById = async (id: string, token: string): Promise<JobRoleDetailedResponse[]> => {
     try {
-        const response: AxiosResponse = await axios.get(URL+id)
+        const response: AxiosResponse = await axios.get(URL+id, getHeader(token))
+
         return response.data;
     } catch (e) {
```
Comment (mlynch147) [inline src/services/JobRoleService.ts:27]: This looks like it is undoing the changes that were made for Job Role Description ??  

Maybe this just needs the latest main branch merged into this feature branch ??

**Review by mlynch147 — CHANGES_REQUESTED** [review]
one comment on JobRoleService.ts

File: src/services/JobRoleService.ts (hunk 1 above)
Comment (TaylorB-softdevkainos) [reply src/services/JobRoleService.ts:27]: Fixed

Comment (mlynch147) [reply src/services/JobRoleService.ts:27]: perfect, thanks...

**Review by ldevlin559 — APPROVED** [review]

**Review by mlynch147 — APPROVED** [review]

**Pull request merged**